source/frontend/src/hooks/layer-api/useWfsLayer.ts

```diff
@@ -67,6 +67,7 @@ export const useWfsLayer = (
       },
       [layerOptions, url],
     ),
+    throwError: true,
```

Comment:
the usewfs layer will bubble up the errors now

Reply:
Did you check all usages of `useWfsLayer` to make sure they handle errors correctly?